ersilia/cli/commands/catalog.py

```diff
@@ -20,11 +20,13 @@ def catalog_cmd():
 
     Examples
     --------
-    List models available in the local computer and show more detailed information:
-    $ ersilia catalog --local --more
+    .. code-block:: console
 
-    Display model card for a specific model ID and show catalog in table format:
-    $ ersilia catalog --card <model_id> --as-table
+        # List models available in the local computer and show more detailed information:
+        $ ersilia catalog --local --more
+
+        # Display model card for a specific model ID and show catalog in table format:
+        $ ersilia catalog --card <model_id> --as-table
```

Comment:
Hey @Abellegese since the catalog command displays the catalog in table format by default now, we can remove this option from the docstring.

Reply:
And we can add another line for the --as_json flag.